types/reactour/index.d.ts

```diff
@@ -33,7 +33,7 @@ export interface ReactourStep {
     /**
      * Position of step content
      */
-    position?: ReactourStepPosition;
+    position?: ReactourStepPosition | Number[];
```

Comment:
It should definitely by `number` (lowercase), not `Number`, but looking at the source code here: https://github.com/elrumordelaluz/reactour/blob/master/src/components/Guide.js (line 65, `pos` function definition, notice `helperPosition` should be a two-item array) it seems like actually the best typing here would be `[number, number]`.

Reply:
I always get the type casing mixed up lol, updated to `[number, number]`, that is the best type since it always expects two numbers for the x and y position if passing an array